src/tests/ftest/launch.py

```diff
@@ -2411,19 +2412,13 @@ def _cleanup_procs(self, test):
 
         proc_pattern = "|".join(PROCS_TO_CLEANUP)
         logger.debug("Looking for running processes: %s", proc_pattern)
-        pgrep_cmd = f"pgrep --list-full '{proc_pattern}'"
-        pgrep_result = run_remote(logger, hosts, pgrep_cmd)
-        if pgrep_result.passed_hosts:
-            any_found = True
-            logger.debug("Killing running processes: %s", proc_pattern)
-            pkill_cmd = f"sudo -n pkill -e --signal KILL '{proc_pattern}'"
-            pkill_result = run_remote(logger, pgrep_result.passed_hosts, pkill_cmd)
-            if pkill_result.failed_hosts:
-                message = f"Failed to kill processes on {pkill_result.failed_hosts}"
-                self._fail_test(self.result.tests[-1], "Process", message)
-            else:
-                message = f"Running processes found on {pgrep_result.passed_hosts}"
-                self._warn_test(self.result.tests[-1], "Process", message)
+        detected, running = stop_processes(logger, hosts, f"'{proc_pattern}'")
```

Comment:
As you asked, it wouldn't hurt to use `force=True` here, but this is already more robust than previous behavior